ui/app/models/task.js

```diff
@@ -79,4 +82,20 @@ export default class Task extends Fragment {
       );
     }
   }
+
+  // TODO: This async fetcher seems like a better fit for most of our use-cases than the above getter (which cannot do async/await)
+  async getPathLinkedVariable() {
+    if (!this._job) {
+      await this._fetchParentJob();
+    }
```

Comment:
There isn't a direct link between jobs and their tasks; instead, there's an intermediary taskGroup. My take is that little helper methods like this are warranted shortcuts.

Reply:
Totally agree. Once upon a time Model Fragments were going to support proper relationships, but alas, here we are.